src/pages/UpdateShelter/UpdateShelter.tsx

```diff
@@ -34,15 +34,19 @@ const UpdateShelter = () => {
   } = useFormik<IUpdateShelter>({
     initialValues: {
       shelteredPeople: shelter.shelteredPeople ?? 0,
+      capacity: shelter.capacity ?? 0,
       petFriendly: shelter.petFriendly ?? false,
+      address: shelter.address ?? '',
     },
     enableReinitialize: true,
     validateOnBlur: false,
     validateOnChange: false,
     validateOnMount: false,
     validationSchema: Yup.object().shape({
       shelteredPeople: Yup.number().required('Este campo deve ser preenchido'),
+      capacity: Yup.number().required('Este campo deve ser preenchido'),
       petFriendly: Yup.bool().required('Este campo deve ser preenchido'),
+      address: Yup.string().required('Este campo deve ser preenchido'),
```

Comment:
Sei que seguiu o fluxo, mas podemos ter mensagens melhores aqui para cada item. Vou deixar uma sugestão abaixo:
```suggestion
      shelteredPeople: Yup.number().required('Informe como está a lotação do abrigo'),
      capacity: Yup.number().required('É necessário informar a capacidade'),
      petFriendly: Yup.bool().required('Informe se o abrigo aceita animais'),
      address: Yup.string().required('É necessário informar o endereço do abrigo'),
```